### ISSUE TITLE
Did you removed option for rounded bar chart??

### ISSUE THREAD

**Issue by jsinhSolanki** [apexcharts/apexcharts.js#2291]
Hi, I am using bar chart. In your previous version of vue-apexcharts I had rounded bar charts. But as I updated to latest version. I lost my rounded corners in my bar chart. I checked your docs I can not find option I used previously.
```
...
endingShape: 'rounded',
...
```

Before(ver 1.2.8):
![sdf](https://user-images.githubusercontent.com/31243533/54921982-2b4b7980-4f2d-11e9-8ed4-926204c042b5.PNG)
Now:
![sdfdf](https://user-images.githubusercontent.com/31243533/54921992-30102d80-4f2d-11e9-9f61-1cc0e87ea8b3.PNG)
I have same code. I havn't changed any parameter in chart options.

~ Regards

**Comment by junedchhipa** [comment]
I am working on a better implementation for border-radius in bar charts.
It will be released along with v3.26.0

**Comment by junedchhipa** [comment]
Released v3.26.0. Apply manual border-radius like below

```
plotOptions: {
  bar: {
    borderRadius: 5
  }
}
```

**Comment by blewell321** [comment]
> Released v3.26.0. Apply manual border-radius like below
> 
> ```
> plotOptions: {
>   bar: {
>     borderRadius: 5
>   }
> }
> ```

Can this be done for pie/donut chart types?

**Comment by manojsethi** [comment]
Can border-radius be applied to top only?

**Comment by everttrollip** [comment]
> Released v3.26.0. Apply manual border-radius like below
> 
> ```
> plotOptions: {
>   bar: {
>     borderRadius: 5
>   }
> }
> ```

@junedchhipa after your update (thank you!) I can see you only select the last item in the series on which to draw the border. Is there an option to add a border to item in the series? Basically the opposite of what @nikitatrifan has asked for, 

> Hey guys, would it be possible to make rounded only the top and bottom parts of a stacked chart instead of making rounded every piece of it? @junedchhipa

**Comment by everttrollip** [comment]
Well, I noticed that I can pass an array to borderRadius, and that then adds a border radius to each series, 

```
plotOptions: {
  bar: {
    borderRadius: [18, 18, 18]
  }
}
```

Is there a way to avoid the overlap? Hovering over the series in the legend shows the radius is correctly applied, but is "overlapping". Can I add a margin right for example? 
<img width="1064" alt="CleanShot 2022-05-03 at 14 51 15" src="https://user-images.githubusercontent.com/32848592/166456339-ad7e8582-b2e5-4b09-b532-05ba918b6fe0.png">
<img width="1070" alt="CleanShot 2022-05-03 at 14 53 29@2x" src="https://user-images.githubusercontent.com/32848592/166456430-08696c5d-38db-4b3a-95f0-19d2f3f9ac6f.png">
